src/Features/LanguageServer/Protocol/Features/Diagnostics/EngineV2/DiagnosticIncrementalAnalyzer_GetDiagnosticsForSpan.cs

```diff
@@ -112,9 +113,17 @@ private sealed class LatestDiagnosticsForSpanGetter
                 var logPerformanceInfo = range.HasValue && blockForData;
                 var compilationWithAnalyzers = await GetOrCreateCompilationWithAnalyzersAsync(document.Project, ideOptions, stateSets, includeSuppressedDiagnostics, cancellationToken).ConfigureAwait(false);
 
+                // If we are computing full document diagnostics, we will attempt to perform incremental
+                // member edit analysis. This analysis is currently only enabled with LSP pull diagnostics.
```

Comment:
> This analysis is currently only enabled with LSP pull diagnostics.

Why?

Reply:
This analysis was re-enabled quite recently and it was decided not to invest in non-LSP space. It would need changes to solution crawler based DiagnosticIncrementalAnalyzer, which would be throwaway work.